vulnerabilities/tests/test_mozilla.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
Could you add add some true unit tests too for each of your functions?

Reply:
OK. Replacing these with true unit tests

Reply:
I meant add unit tests, not replace these entirely. Both are useful